src/ZODB/serialize.py

```diff
@@ -680,7 +680,7 @@ def get_refs(a_pickle):
     u.noload()
 
     # Now we have a list of references.  Need to convert to list of
-    # oids and class info:
+    # oids and class info(object):
```

Comment:
Thanks for finding that!

Unfortunately the other regex misses not just the things that are in docstrings/doctests (which aren't important for performance, granted, but are important for consistency) but it also missed a handful of classes (E.g [this one](https://github.com/zopefoundation/ZODB/pull/160/files#diff-e373c22d07eb1217851d5356905a8f5eR115)) for some reason.